frontend/src/static/js/v_summary.js

```diff
@@ -98,16 +98,27 @@ window.vSummary = {
     },
 
     filterGroupSelection() {
+      this.getFiltered();
```

Comment:
why is this moved to the top?

Reply:
Because to `resetMergedGroupsMap()` the info of repo names is needed (to populate the map).

Reply:
I see